.cz-config.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+    types: [
+        { value: 'feat', name: 'feat:     A new feature' },
+        { value: 'fix', name: 'fix:      A bug fix' },
+        { value: 'WIP', name: 'WIP:      Work in progress' },
+        {
+            value: 'chore',
+            name:
+                'chore:    Changes to the build process or auxiliary tools\n            and libraries such as documentation generation',
+        },
+        { value: 'docs', name: 'docs:     Documentation only changes' },
+        {
+            value: 'style',
+            name:
+                'style:    Changes that do not affect the meaning of the code\n            (white-space, formatting, missing semi-colons, etc)',
+        },
+        {
+            value: 'refactor',
```

Comment:
Maybe we should add this rule to semantic-release, so it triggers a "patch"(fix) build.

Reply:
I see there is 'refactor' in semantic release config https://github.com/RedHatInsights/patchman-ui/blob/master/package.json#L137. Am I looking at the right place? I am not very experienced with semantic release config :)